src/components/alert/alert.css

```diff
@@ -30,5 +30,6 @@
     align-items: center;
     flex-direction: row;
     justify-content: center;
+    z-index: 999;
 
```

Comment:
try to give it below 10 .. best practise

Reply:
read about stacking context otherwise this can give you living hell (I came from that because of using this z-index :100)